concepticondata/conceptlists/Capell-1956-558.tsv

```diff
@@ -0,0 +1,559 @@
+Capell-1956-558-52	175	BLOW (OF WIND)	52	blow (of wind)	Capell1956[13]
+Capell-1956-558-53	176	BLOW (WITH MOUTH)	53	blow (with mouth)	Capell1956[14]
+Capell-1956-558-54	379	BLUNT	54	blunt	Capell1956[14]
+Capell-1956-558-55	1480	BODY	55	body	Capell1956[14]
+Capell-1956-558-56	1145	SWAMP	56	bog	Capell1956[14]
+Capell-1956-558-57	1394	BONE	57	bone	Capell1956[15]
+Capell-1956-558-58	135	BOOMERANG	58	boomerang	Capell1956[15]
+Capell-1956-558-59	1090	BE BORN	59	be born	Capell1956[15]
+Capell-1956-558-60	1321	BOTTLE	60	bottle	Capell1956[15]
+Capell-1956-558-61	1366	BOY	61	boy	Capell1956[16]
+Capell-1956-558-62	3020	BREAK (BREAKING)	62	break	Capell1956[16]
+Capell-1956-558-63	1407	BREATHE	63	breathe	Capell1956[16]
+Capell-1956-558-64	3982	BREEZE	64	breeze	Capell1956[16]
+Capell-1956-558-65	692	BRING	65	bring	Capell1956[17]
+Capell-1956-558-66	588	BROKEN	66	broken	Capell1956[17]
+Capell-1956-558-67	1760	YOUNGER BROTHER	67	brother (younger)	Capell1956[17]
+Capell-1956-558-68	1759	OLDER BROTHER	68	brother (elder)	Capell1956[17]
+Capell-1956-558-69	934	BUFFALO	69	buffalo	Capell1956[18]
+Capell-1956-558-70			70	bullroarer	Capell1956[18]
+Capell-1956-558-71	2102	BURN	71	burn	Capell1956[18]
+Capell-1956-558-72			72	burrawang (cycas) fruit	Capell1956[18]
+Capell-1956-558-73	2949	LAIR	73	burrow	Capell1956[19]
+Capell-1956-558-74	2811	BUSHFIRE	74	bush-fire	Capell1956[19]
+Capell-1956-558-75	1791	BUTTERFLY	75	butterfly	Capell1956[19]
+Capell-1956-558-76	94	GRADUALLY	76	by and by	Capell1956[19]
+Capell-1956-558-77	663	BAY	77	bay	Capell1956[20]
+Capell-1956-558-78	1531	BRANCH	78	branch	Capell1956[20]
+Capell-1956-558-79	1084	CALL	79	call	Capell1956[20]
+Capell-1956-558-80	1105	CAMP	80	camp	Capell1956[20]
+Capell-1956-558-81	1051	CAN	81	can	Capell1956[21]
+Capell-1956-558-82	1970	CANOE	82	canoe	Capell1956[21]
+Capell-1956-558-83	1662	CAPE	83	cape	Capell1956[21]
+Capell-1956-558-84			84	capsize	Capell1956[21]
+Capell-1956-558-85	700	CARRY	85	carry	Capell1956[22]
+Capell-1956-558-86	702	CATCH	86	catch	Capell1956[22]
+Capell-1956-558-87	2247	CATERPILLAR	87	caterpillar	Capell1956[22]
+Capell-1956-558-88	3777	CATFISH	88	catfish	Capell1956[22]
+Capell-1956-558-89	1155	CAVE	89	cave	Capell1956[23]
+Capell-1956-558-90	1974	CENTIPEDE	90	centipede	Capell1956[23]
+Capell-1956-558-91	1188	CHARCOAL	91	charcoal	Capell1956[23]
+Capell-1956-558-92	1730	CHEEK	92	cheek	Capell1956[23]
+Capell-1956-558-93	1592	CHEST	93	chest	Capell1956[24]
+Capell-1956-558-94	1510	CHIN	94	chin	Capell1956[24]
+Capell-1956-558-95			95	chiton	Capell1956[24]
+Capell-1956-558-96	497	SCAR	96	cicatrix	Capell1956[24]
+Capell-1956-558-97	393	CIRCUMCISION	97	circumcision	Capell1956[25]
+Capell-1956-558-98	1399	CLAY	98	clay	Capell1956[25]
+Capell-1956-558-99	704	CLEAN	99	clean	Capell1956[25]
+Capell-1956-558-100	1310	CLEVER	100	clever	Capell1956[25]
+Capell-1956-558-101	1102	CLIMB	101	climb	Capell1956[26]
+Capell-1956-558-102	1046	SHUT	102	close (adj.)	Capell1956[26]
```

Comment:
SHUT refers to the action, but the entry seems to be quite specific about the adjectival/stative meaning